addons/mail/static/src/core/common/discuss_app_model.js

```diff
@@ -47,8 +21,12 @@ export class DiscussApp extends Record {
     chatWindows = Record.many("ChatWindow");
     isActive = false;
     thread = Record.one("Thread");
-    channels = Record.one("DiscussAppCategory");
-    chats = Record.one("DiscussAppCategory");
+    channels = Record.one("DiscussAppCategory", {
+        compute: () => ({ id: "chats" }),
+    });
+    chats = Record.one("DiscussAppCategory", {
+        compute: () => ({ id: "channels" }),
+    });
```

Comment:
As I said in previous comment, it feels more natural that all data of category are provided here, not in `ThreadModel` relational field with discuss category